users/script.js

```diff
@@ -474,18 +485,76 @@ function getCheckedValues(groupName) {
 
 function getFilteredUsersURL(checkedValuesSkills, checkedValuesAvailability) {
   const params = new URLSearchParams();
-
   checkedValuesSkills.forEach((skill) => {
     params.append('tagId', skill);
   });
-
   checkedValuesAvailability.forEach((availability) => {
     params.append('state', availability);
   });
-
   return `?${params.toString()}`;
 }
 
+function updateQueryParamsToURL(constructedQueryParam) {
```

Comment:
- should we not name this `setQueryParams` / `addQueryParams` as we are setting the query params in the URL?